src/main/java/org/zalando/nakadi/controller/EventPublishingController.java

```diff
@@ -99,6 +104,21 @@ private ResponseEntity postEventInternal(final String eventTypeName,
         }
     }
 
+    private void reportMetrics(final EventTypeMetrics eventTypeMetrics, final EventPublishResult result,
+                               final String eventsAsString, final int eventCount) {
+        if (result.getStatus() == EventPublishingStatus.SUBMITTED) {
+            eventTypeMetrics.reportSizing(eventCount, eventsAsString.getBytes(StandardCharsets.UTF_8).length - 2);
```

Comment:
@lmontrieux I don't think it's going to be that easy. You need to be aware that there are commas in the events themselves. So you would be required to actually "parse" the string to identify these comma separators.